src/TestUtils/src/DeviceTests/AssertionExtensions.iOS.cs

```diff
@@ -528,5 +529,50 @@ public static UIView GetAccessiblePlatformView(this UIView platformView)
 
 			return platformView;
 		}
+
+		public static void TapBackButton(this UINavigationBar uINavigationBar)
+		{
+			var item = uINavigationBar.FindDescendantView<UIView>(result =>
+			{
+				return result.Class.Name?.Contains("UIButtonBarButton", StringComparison.OrdinalIgnoreCase) == true;
+			});
+
+			_ = item ?? throw new Exception("Unable to locate back button view");
+
+			var recognizer = item!.GestureRecognizers!.OfType<UITapGestureRecognizer>().FirstOrDefault();
+			_ = recognizer ?? throw new Exception("Unable to Back Button TapGestureRecognizer");
+
+			recognizer.State = UIGestureRecognizerState.Ended;
```

Comment:
Is this method trying to get the back button or literally just tapping the back button? If the latter, is this state setting all we need?